projects/plugins/jetpack/package.json

```diff
@@ -180,14 +180,17 @@
 		"eslint": "7.31.0",
 		"fs-extra": "10.0.0",
 		"glob": "7.1.6",
+		"install": "0.13.0",
 		"jest": "27.0.4",
 		"jest-environment-jsdom": "^27.0.6",
 		"jetpack-js-test-runner": "workspace:*",
 		"lodash": "4.17.21",
 		"mocha": "8.2.1",
 		"mockery": "2.1.0",
 		"nock": "13.0.7",
+		"node-polyfill-webpack-plugin": "1.1.4",
 		"node-wp-i18n": "1.2.4",
+		"npm": "7.20.6",
```

Comment:
Even more, what's this for??

Reply:
Yeah, weird. I didn't put it there... or at least, I don't _remember_ putting it there.